rstudio/c9s-python-3.11/run-rstudio.sh

```diff
@@ -25,7 +25,11 @@ do
 done  
 # rstudio terminal cant see environment variables set by the container runtime
 # (which breaks kubectl, to fix this we store the KUBERNETES_* env vars in Renviron.site)
+# Also, we store proxy-related env vars lowercased by key so RStudio projects work with proxy by default
 env | grep KUBERNETES_ >> /usr/lib64/R/etc/Renviron.site
+env | grep HTTP_PROXY | tr '[:upper:]' '[:lower:]' >> /usr/lib64/R/etc/Renviron.site
```

Comment:
yeah, not a bad idea. Since for K8S related env vars, they start with KUBERNETES_, too, same would apply for those as well, i.e. making use of `^KUBERNETES_` to indicate begining of line for those, too.